src/pages/index.js

```diff
@@ -15,7 +15,7 @@ const IndexPage = () => {
   const onModalClose = () => {
     dispatch(setHasSeenIntroModal(true))
     setOpenModal(false)
-    window.location.pathname = window.location.pathname + "/recipes"
+    window.location.pathname = window.location.pathname + "recipes"
```

Comment:
does this have to change? concerned it might not work when it's published

Reply:
it doesn't work rn on master

Reply:
really

Reply:
I don't think it's gonna work when its deplloyed though bc it'll end up w 2 slashes

Reply:
idk we can try lol

Reply:
oh you're write nvm